### PR TITLE
Add The Weekly Iteration to the feed sources

### PR DESCRIPTION
**Pull request by croaky**
The Weekly Iteration from Upcase contains wonderful content such as
https://upcase.com/videos/intro-to-docker
and is published weekly,
making it a great fit for the weekly newsletter.

### CODE REVIEW

**Comment by bernerdschaefer** [comment]
Nice! :+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ To test the `rss` package, run `go test ./...`.
 
 Make your change, with new passing tests.
 
+See the change in a web browser by
+installing the package with `go install` and
+running the server with `foreman start`.
```
Comment (calebthompson) [inline CONTRIBUTING.md]: Foreman is pretty foreign (hehe) to Go. I'd say just sticking with `go run main.go` is ideal here.

**Comment by calebthompson** [comment]
One recommendation, :flags: otherwise.

File: CONTRIBUTING.md (hunk 1 above)
Comment (croaky) [inline CONTRIBUTING.md]: Nice, eliminates the `install` step, too.

**Pull request merged**